dev/jenkins/build.sh

```diff
@@ -93,7 +92,8 @@ if [ "$RUN_MAVEN" == "true" ]; then
     mvn $(echo "${ALLUXIO_SONAR_ARGS}") sonar:sonar
   fi
 else
-  echo "RUN_MAVEN was not set to true, skipping maven check"
+  echo "RUN_MAVEN was not set to true, skipping full maven check and only running license check"
+  mvn -Duser.home=/home/jenkins license:check
```

Comment:
is there anything other than license check that we need?

Reply:
license check has been the one that's slipped passed the PR build and caused post-merge failures. i don't recall any other type of failure that has slipped through similarly